types/mini-css-extract-plugin/mini-css-extract-plugin-tests.ts

```diff
@@ -3,6 +3,14 @@ import MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 let configuration: webpack.Configuration;
 
+let loaderOptions: MiniCssExtractPlugin.LoaderOptions = {
```

Comment:
```suggestion
const loaderOptions: MiniCssExtractPlugin.LoaderOptions = {
```

Reply:
You can run 'npm test mini-css-extract-plugin' to preflight your changes locally, btw. thx1

Reply:
Thanks for making this suggestion! I just realized I forgot to use const like you suggested earlier. Sorry about that.

About local testing, I've been trying to do just that, but I keep getting errors about missing types for the package `source-map`, which is referenced in the webpack typings. Do you know why I am getting those errors? I attached an image of the errors early in this PR thread.